src/components/NavBar.jsx

```diff
@@ -2,7 +2,7 @@ import useMoviesContext from "../context/MoviesContext";
 
 export const Tabs = Object.freeze({
   Movies: "Movies",
-  Favourites: "Favorites",
+  Favourites: "Favorits",
```

Comment:
https://github.com/nfeagh/react-flask-app/blob/main/inbox/email_J_10.md Can you please also update this as well as it is no longer to do with American spelling.

Reply:
ah actually that's a good point I forgot about this...
Ok let's leave your email in, let me back this one out and we can discuss with Jeenal on Tuesday.